internal/predicate/compositebackup_predicate.go

```diff
@@ -51,6 +52,8 @@ func (p CompositeBackupPredicate) Update(evt event.UpdateEvent) bool {
 		return p.NonAdminBackupPredicate.Update(p.Context, evt)
 	case *velerov1.Backup:
 		return p.VeleroBackupQueuePredicate.Update(p.Context, evt) || p.VeleroBackupPredicate.Update(p.Context, evt)
+	case *velerov1.PodVolumeBackup:
+		return p.VeleroPodVolumeBackupPredicate.Update(p.Context, evt)
```

Comment:
for DataUpload/DataDownload I think I do not need a new predicate, but for PodVolumeBackup/PodVolumeRestore we can not run away from this, right?